index.html

```diff
@@ -40,13 +40,274 @@ <h3 class="large-text"> I can help you build a product, feature or website. Look
 don’t hesitate to contact me.</h3>
         <br>
         <ul class="social-links">
-          <li><a class="no-style" href="https://twitter.com/SiriMoon11"><i class="fab fa-twitter"></i></a></li>
-          <li><a class="no-style" href="https://www.linkedin.com/in/melanie-arellano-92aaa9194/"><i class="fab fa-linkedin-in"></i></a></li>
-          <li><a class="no-style" href="#"><i class="fab fa-medium-m"></i></a></li>
-          <li><a class="no-style" href="https://github.com/melaniesigrid"><i class="fab fa-github-alt"></i></a></li>
-          <li><a class="no-style" href="#"><i class="fab fa-angellist"></i></a></li>
+          <li><a class="yes-style" href="https://twitter.com/SiriMoon11"><i class="fab fa-twitter"></i></a></li>
+          <li><a class="yes-style" href="https://www.linkedin.com/in/melanie-arellano-92aaa9194/"><i class="fab fa-linkedin-in"></i></a></li>
+          <li><a class="yes-style" href="#"><i class="fab fa-medium-m"></i></a></li>
+          <li><a class="yes-style" href="https://github.com/melaniesigrid"><i class="fab fa-github-alt"></i></a></li>
+          <li><a class="yes-style" href="#"><i class="fab fa-angellist"></i></a></li>
         </ul>
       </section>
+        <br>
+        <br>
+        <div class="chevron-container">
+          <a href="#works" class="chevron-button">
+            <i class="fas fa-chevron-down downward"></i>
+          </a>
+        </div>
+      <section class="works" id="works">
+        <br>
+        <h2 class="works-title">My Recent<br>Works</h2>
+        <div class="separator">
+          <hr class="rounded">
+        </div>
+        <br>
+        <br>
+        <div class="card-container">
+          <div class="card">
+            <div class="cardlet">
+              <h3 class="project-title">Multi-Post Stories<br>Gain+Glory</h3>
+              <div class="languages-container">
+                <div class="language-box" id="ruby">
+                  <h4 class="language-text">Ruby on rails</h4>
+                </div>
+                <div class="language-box" id="css">
+                  <h4 class="language-text">css</h4>
+                </div>
+                <div class="language-box" id="js">
+                  <h4 class="language-text">JavScript</h4>
+                </div>
+                <div class="language-box" id="html">
+                  <h4 class="language-text">html</h4>
+                </div>
+              </div>
```

Comment:
- [ ] I would suggest using the `<ul>` tag instead of the `<div>` tag, this is a better option for listing items for a semantically correct page :wink:
Don't forget to wrap each list item inside a `<li>` tag :memo: